spk/synocli-net/Makefile

```diff
@@ -17,7 +17,7 @@ MAINTAINER = ymartin59
 DESCRIPTION = "SynoCli Network Tools provides a set of small command-line utilities: screen, tmux, mosh, socat, nmap, sshfs, fritzctl. Credits to Sebastian Schmidt \(publicarray\) for icons"
 DISPLAY_NAME = SynoCli Network Tools
 STARTABLE = no
-CHANGELOG = "3. Build sshfs 2.10 with fuse updated to v2.9.9"
+CHANGELOG = "built with go 1.13.1"
```

Comment:
only fritzctl is built with go

Reply:
Changed in https://github.com/SynoCommunity/spksrc/pull/3779/commits/c0cd05c187bb78a5088bd0ce9895ae238bd78e07